invitebot.go

```diff
@@ -40,10 +40,6 @@ func onBotInit(cli *botcli.BotCli, bot *deltachat.Bot, cmd *cobra.Command, args
 			if err != nil {
 				cli.Logger.Error(err)
 			}
-			err = bot.Rpc.SetConfig(accId, "delete_device_after", option.Some("1800"))
-			if err != nil {
-				cli.Logger.Error(err)
-			}
```

Comment:
The bot needs to save its messages locally for this feature to work, so I guess this is a dealbreaker for merging it into main. I guess I will fork this instead.